openreview/venue/matching.py

```diff
@@ -1165,6 +1165,23 @@ def deploy_assignments(self, assignment_title, overwrite):
             reviewer_name = venue.area_chairs_name
             review_name = 'Meta_Review'
 
+        # Remove reviewers_proposed_assignment_title if deploying reviewer assignments
+        if reviewer_name == venue.reviewers_name:
```

Comment:
I think it is better to use `self.is_reviewer` to check if we are deploying reviewer assignments